home/.chezmoi.yaml.tmpl

```diff
@@ -140,6 +140,11 @@ data:
     docker:
       doRegion: nyc1
     domain: "{{ $domain }}"
+    gitlab:
+      glurl: "https://gitlab.com/"
+      runnerDescription: "Docker executor"
+      runnerImage: "alpine:latest"
+      runnerTags: ""
```

Comment:
Please get rid of runnerTags and instead automatically add tags like $HOST, the operating system type, relevant useful information.

Reply:
As you know, tags are the only way to configure a job to run on a specific runner. So, providing a generic tag like the host name may not always help identify what tools/software are available in a runner. Tags also allow having multiple runner where each one contains a different version of a program, and that can help in running builds with different environments.

The script is designed to either create the runner with tags or mark them to pick _untagged_ jobs. While it is possible to have tagged runners pick untagged jobs, I made it this way to introduce a level of  'isolation' so separate runners are used for untagged jobs. We can change this logic, may be introduce an input variable to control whether a given runner should pick untagged jobs.

If you prefer removing this input, we can do that as well. Please let me know.

Reply:
good point, let's tag them with:

1. The hostname
2. `docker` if Docker is installed
3. `virtualbox` if VirtualBox is installed

I would like to abstract as much as the config as necessary. We can keep the runnerTags here if you think its beneficial.

Reply:
Added these tags and updated the description.